weheat/abstractions/heat_pump.py

```diff
@@ -154,5 +198,9 @@ def heat_pump_state(self) -> State:
             return self.State.MANUAL_CONTROL
         elif numeric_state == 200:
             return self.State.DEFROSTING
-        else:
-            return self.State.UNDEFINED
+        return None
+
+    @property
+    def energy_total(self):
+        """The total used energy in kWh."""
```

Comment:
from 2024 YTD? in line with comment at the top.

Reply:
it will be from 2023 onwards with no end to get the absolute total energy.